src/main/pg/sdio.h

```diff
@@ -20,13 +20,25 @@
 
 #pragma once
 
+#include "drivers/io_types.h"
 #include "pg/pg.h"
 
 typedef struct sdioConfig_s {
     uint8_t clockBypass;
     uint8_t useCache;
     uint8_t use4BitWidth;
     int8_t dmaopt;
+
+#ifdef STM32H7
```

Comment:
Conditionals can not be put into a parameter group, or the contract of a given parameter group of a given version being well-defined is broken.
This should go into a separate parameter group like `sdioPinConfig`.

Reply:
👍